pkg/virt-controller/watch/clone/clone.go

```diff
@@ -346,10 +346,14 @@ func (ctrl *VMCloneController) updateStatus(origClone *clonev1alpha1.VirtualMach
 		}
 
 		if syncInfo.targetVMCreated {
+			assignPhase(clonev1alpha1.WaitingForTargetVMToBeReady)
+		}
+	}
+	if isInPhase(vmClone, clonev1alpha1.WaitingForTargetVMToBeReady) {
+		if syncInfo.targetVMReady {
 			vmClone.Status.SnapshotName = nil
 			vmClone.Status.RestoreName = nil
-			assignPhase(clonev1alpha1.Succeeded)
-
+			assignPhase(clonev1alpha1.WaitingForTargetVMToBeReady)
```

Comment:
this should have stayed succeeded isnt it?

Reply:
:+1: